scripts/build/electron-installer-redhat-linux.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+###
+# Copyright 2016 resin.io
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+set -u
+set -e
+
+OS=$(uname)
+if [[ "$OS" != "Linux" ]]; then
+  echo "This script is only meant to be run in GNU/Linux" 1>&2
+  exit 1
+fi
+
+ELECTRON_INSTALLER_REDHAT="./node_modules/.bin/electron-installer-redhat"
```

Comment:
Whoops, my mistake, that now causes CI errors in `npm-shrinkwrap.json`.
Can you rebase this PR off the latest version of `master` (I'm afraid there's been quite a few changes to the Etcher codebase in the build-up to the 1.0.0 release), and then move `electron-installer-redhat` from `package.json` to the Dockerfile templates, as was done for `electron-installer-debian` in #1379 ?

If that gets too complicated, then just let me know and I can make those changes for you.